end-to-end/src/test/java/nl/tudelft/wdm/group1/endToEnd/EndToEndTest.java

```diff
@@ -1,20 +1,21 @@
 package nl.tudelft.wdm.group1.endToEnd;
```

Comment:
Unrelated, but could you rename the package to `nl.tudelft.wdm.group1.endtoend`? 😅